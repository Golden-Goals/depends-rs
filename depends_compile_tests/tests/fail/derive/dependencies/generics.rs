use depends::derives::Dependencies;

#[derive(Dependencies)]
struct Bar<A, B> {
    node1: A,
    node2: B,
}

fn main() {}
