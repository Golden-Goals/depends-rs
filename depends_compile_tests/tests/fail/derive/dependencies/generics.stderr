error: Dependencies can't be defined with generics.
 --> tests/fail/derive/dependencies/generics.rs:4:11
  |
4 | struct Bar<A, B> {
  |           ^
